server/services/v1/query_runner.go

```diff
@@ -152,6 +154,22 @@ func (runner *BaseQueryRunner) GetCollections(db *metadata.Database, collName st
 	return collection, nil
 }
 
+func IsNull(tp schema.FieldType, val []byte) bool {
+	switch tp {
+	case schema.Int32Type:
+		return string(val) == "0"
+	case schema.Int64Type:
+		return string(val) == "0"
+	case schema.UUIDType:
+		return string(val) == uuid.UUID{}.String()
+	case schema.DateTimeType:
+		return string(val) == time.Time{}.Format(time.RFC3339Nano)
```

Comment:
do we need format here? just time.Time{}?

Reply:
We need to, time is a raw string in the json document here.